CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Allow null arguments on query. ([#872](https://github.com/realm/realm-dart/pull/872)). Fixes [#871](https://github.com/realm/realm-dart/issues/871)
 * Previously removeAt did not truncate length. ([#884](https://github.com/realm/realm-dart/pull/884)). Fixes [#883](https://github.com/realm/realm-dart/issues/883)
 * List.length= now throws, if you try to increase length, ([#894](https://github.com/realm/realm-dart/pull/894)).
+* Queries on results was broken. ([#900](https://github.com/realm/realm-dart/pull/900)). Fixes issue [#908](https://github.com/realm/realm-dart/issues/908)
```

Comment:
That's way too vague. Clearly they were working for a range of cases, otherwise our tests wouldn't have passed.

Reply:
Can't interested parties just follow the link to the issue?

Reply:
Anyway, I have clarified:
```markdown
* Queries on results didn't filter the existing results. Fixes issue [#908](https://github.com/realm/realm-dart/issues/908)
```

Reply:
I have added the example from the issue in the CHANGELOG, so that people won't need to press a link in the browser they are using to read the same.

Reply:
Thank you. I want to clarify I'm not arbitrarily asking for this and I'd be happy to discuss the level of detail we aim for in changelog entries if you think that'll be valuable.